### PR TITLE
Eu opinions download FE

### PR DESCRIPTION
**Pull request by defaye**
Add new filter "eu_decision_filter" options:
 * "default" for "EU Decision Types"
 * "in-consultation"

Add helper classes .p & .m for padding & margin
Add helper class .border for bordering elements
Add helper classes prefixed with .bs- for grid templating
Clear years from filters when "current" is selected

### CODE REVIEW

File: app/assets/javascripts/species/controllers/downloads_for_eu_decisions_controller.js.coffee [hunk 1]
```diff
@@ -74,26 +81,35 @@ Species.DownloadsForEuDecisionsController = Ember.Controller.extend
...
-          {
-            positiveOpinions: @get('positiveOpinions')
-            negativeOpinions: @get('negativeOpinions')
-            noOpinions: @get('noOpinions')
-            srgReferral: @get('srgReferral')
-            suspensions: @get('suspensions')
-          }
-        csv_separator: @get('controllers.downloads.csvSeparator')
+        decision_types: {
+          negativeOpinions: @get('negativeOpinions')
```
Comment (defaye) [inline app/assets/javascripts/species/controllers/downloads_for_eu_decisions_controller.js.coffee:93]: Arguably these should also be in snake_case but as it depends on the BE implementation I will leave it be

File: app/assets/javascripts/species/views/downloads/downloads_popup.js.coffee [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Species.DownloadsPopup = Ember.View.extend
   templateName: 'species/downloads_popup'
-  classNames: ['dwonload-block']
+  classNames: ['download-block']
```
Comment (defaye) [inline app/assets/javascripts/species/views/downloads/downloads_popup.js.coffee:3]: Typo that has gone unnoticed has been corrected

**Comment by lucacug** [comment]
@defaye  everything looks good, the only thing I would change is to avoid sending the params to the BE related to the eu_decision_types when the "In consultation" radio button is selected. I'm managing this in the BE, but I think it is cleaner to do it on the FE as well. What do you think?

**Comment by defaye** [comment]
> @defaye everything looks good, the only thing I would change is to avoid sending the params to the BE related to the eu_decision_types when the "In consultation" radio button is selected. I'm managing this in the BE, but I think it is cleaner to do it on the FE as well. What do you think?

As per our conversation, yes can absolutely do this and it was actually something I wanted to do in the current commit so more than happy to add this in

**Comment by lucacug** [comment]
looks good. Merging!

**Pull request merged**